deployment/v3/utils/prop_migrator/file_comparator.py

```diff
@@ -0,0 +1,135 @@
+import os
+import subprocess
+import csv
+
+# Prompt for existing Old configuration repo URL and branch
+old_repo_url = input("Enter existing old configuration repo URL: ")
+old_repo_branch = input("Enter existing old configuration repo branch: ")
+
+# Create Old dir
+old_dir = "old-config"
+os.makedirs(old_dir, exist_ok=True)
+
+# Clone the existing repo with Old-repo-branch inside the old directory
+old_repo_path = os.path.join(old_dir, old_repo_branch)
+subprocess.run(["git", "clone", "--branch", old_repo_branch, old_repo_url, old_repo_path])
+
+# Prompt for New based configurations URL and branch
+new_repo_url = input("Enter new based configurations URL: ")
+new_repo_branch = input("Enter new based configuration branch: ")
+
+# Create New dir
+new_dir = "new-config"
+os.makedirs(new_dir, exist_ok=True)
+
+# Clone the New repo with New-repo-branch inside the New directory
+new_repo_path = os.path.join(new_dir, new_repo_branch)
+subprocess.run(["git", "clone", "--branch", new_repo_branch, new_repo_url, new_repo_path])
+
+# Retrieve property file pairs from CSV
+property_file_pairs = []
+csv_file = "property_file_pairs.csv"
+with open(csv_file, 'r') as file:
+    reader = csv.reader(file)
+    for row in reader:
+        property_file_pairs.append(row)
+
+# Compare property files and create different_properties.csv, file1_only.csv, and file2_only.csv
+different_properties = []
+old_only_properties = []
+new_only_properties = []
+
+for old_filename, new_filename in property_file_pairs:
+    old_file = os.path.join(old_repo_path, "sandbox", old_filename)
```

Comment:
path after mosip-config or cloned repo should be part of propert_file_mapping.csv.
And this file name creation should be removed.